pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -127,6 +127,24 @@ func (a *Bootstrap) Generate(dependencies asset.Parents) error {
 	if err != nil {
 		return err
 	}
+
+	// Check for optional platform specific directories
+	platform := installConfig.Config.Platform.Name()
+	platformFilePath := fmt.Sprintf("bootstrap/%s/files", platform)
+	if _, err := os.Stat(platformFilePath); !os.IsNotExist(err) {
+		err = a.addStorageFiles("/", platformFilePath, templateData)
+		if err != nil {
+			return err
+		}
+	}
+	platformUnitPath := fmt.Sprintf("bootstrap/%s/units", platform)
```

Comment:
This should be `bootstrap/%s/systemd/units`.

Reply:
Oops, thanks!